project/src/main/java/com/google/autograder/servlets/ListAssignmentSubmissionsServlet.java

```diff
@@ -0,0 +1,24 @@
+package com.google.autograder.servlets;
+
+import java.net.URL;
+import java.io.IOException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Entity;
+import com.google.autograder.servlets.helpers.Services;
+import com.google.appengine.api.datastore.Query.Filter;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query.FilterOperator;
+import com.google.appengine.api.datastore.Query.FilterPredicate;
+
+@WebServlet("/listAssignmentSubmissions")
+public final class ListAssignmentSubmissionsServlet extends HttpServlet {
```

Comment:
If this isn't used right now, this should ideally go in a separate PR

Reply:
Alright no problem, I'll make sure not to overlap work in commits going forward.